src/Console/OutputFormatter/AbstractOutputFormatter.php

```diff
@@ -44,6 +44,9 @@
  */
 abstract class AbstractOutputFormatter implements OutputFormatter
 {
+    /**
+     * @var int
```

Comment:
I don't get it: it has a default value no? Isn't that enough to infer the type?

Reply:
Default value isn't enough to infer a type, it needs to be set in the constructor for that.